lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -115,6 +119,8 @@ public function __construct(Configuration $configuration, $version, $class)
         $this->platform = $this->connection->getDatabasePlatform();
         $this->migration = new $class($this);
         $this->version = $version;
+        $schemaManipulator = new SchemaDiffProvider($this->sm, $this->platform);
```

Comment:
I was wondering if removing the private property would be an BC break. I think not and if not then we can safely remove it.